Client/src/components/common/Input/index.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
-import './index.css';
 
-const Input = props =>
-  <input  {...props}  className='input' />;
-  
-export default Input;
+const Input = props => {
+  const {name, placeholder, id, onChange, inputClassName} = props
+  return (
+    <input name={name} placeholder={placeholder} id={id} onChange={onChange} className={inputClassName} />
+  )
```

Comment:
I think 
```
const Input = props =>	const Input = props => {
  <input  {...props}  className='input' />;
```
usefull and  we can reuse it

Reply:
see the comment above why is this not good and the component is already useable !!!!!